File: labs/OOP/Lab_06/RowsLibrary/Numbers.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RowsLibrary
{
    public class Numbers : Rows
    {
        public Numbers(string row) : base(row)
        {
        }
        public override int RowLength()
        {
            length = row.Length;
            return length;
        }
        public override string ChangeSymbol()
        {
            row = row.Replace("3", "11");
            return row;
        }
    }
}

File: labs/OOP/Lab_06/Lab_6/Numbers.cpp
#include "Numbers.h"


int Numbers::RowLength()
    {
        length = strlen(row);
        return length;
    }
char* Numbers::ChangeSymbol() 
    {
        char y = '3';
        char x = '1';
        for (int i = 0; i < strlen(row); i++)
        {
            if (row[i] == y) {
                row[i] = x;
            }
        }
        return row;
    }
